web/app/js/components/CallToAction.jsx

```diff
@@ -38,6 +38,7 @@ const CallToAction = ({ resource, numResources }) => {
       <Typography>The service mesh was successfully installed!</Typography>
       <Stepper
         activeStep={lastStep}
+        className={classes.instructions}
```

Comment:
This is a great fix, thanks! It looks like there are still two un-used `styles` specifications, `root` and `button`. Would you mind removing those from the component as well before we merge this in?

Reply:
Yes sure